pygfunction/gfunction.py

```diff
@@ -282,6 +287,75 @@ def __init__(self, boreholes_or_network, alpha, time=None,
         if self.time is not None:
             self.gFunc = self.evaluate_g_function(self.time)
 
+    @classmethod
+    def from_static_params(cls,
+                           H: npt.ArrayLike,
+                           D: npt.ArrayLike,
+                           r_b: npt.ArrayLike,
+                           x: npt.ArrayLike,
+                           y: npt.ArrayLike,
+                           alpha: float,
+                           time: npt.ArrayLike = None,
+                           method: str = 'equivalent',
+                           m_flow_borehole: float = None,
+                           m_flow_network: float = None,
+                           cp_f=None,
+                           options={},
+                           tilt: npt.ArrayLike = 0.,
+                           orientation: npt.ArrayLike = 0.,
+                           boundary_condition: str = 'UHTR',
```

Comment:
I would suggest 'MIFT' as the default value. It seems coherent to me to evaluate this g-function if all pipe parameters are provided.